src/components/NFTPortfolio/Item.vue

```diff
@@ -74,8 +77,24 @@ export default {
     };
   },
   computed: {
+    ownedNftId() {
+      return this.collectorMap?.[this.getAddress]?.[0];
+    },
```

Comment:
Seems there is a `nftIdCollectedFirstByUser` in nft mixin that we can resue

Reply:
[♻️ Use mixin data instead of new computed props](https://github.com/likecoin/liker-land/pull/1889/commits/f48701ae74a7ff383a0a5cb456641b7d2a5d3599)